### ISSUE TITLE
[HTML] html-in-canvas

### ISSUE THREAD

**Issue by progers** [WebKit/standards-positions#630]
### WebKittens

@annevk, @smfr, @shallawa, @cookiecrook 

### Title of the proposal

html-in-canvas

### URL to the spec

https://html.spec.whatwg.org/

### URL to the spec's repository

https://github.com/WICG/html-in-canvas

### TAG Design Review URL

https://github.com/w3ctag/design-reviews/issues/1204

### Mozilla standards-positions issue URL

https://github.com/mozilla/standards-positions/issues/1076

### Description

A fundamental capability missing from the web is the ability to complement `<canvas>` with HTML. Adding this capability enables `<canvas>` surfaces to benefit from features of browsers (text layout, interactivity, accessibility, etc).

This was most recently proposed as canvas place element (https://github.com/WebKit/standards-positions/issues/403), and a [concern](https://github.com/WICG/html-in-canvas/issues/15#issuecomment-3576980621) raised was around retained-mode canvas. This has been addressed by returning to an immediate-mode API. [This comment](https://github.com/whatwg/html/issues/10650#issuecomment-4001293759) outlines the major changes, and the explainer (https://github.com/WICG/html-in-canvas) has been updated with all of the details.

**Comment by lukewarlow** [comment]
https://github.com/whatwg/html/issues/10650 - fyi this is now going through the HTML stages process.

So I'll remove the venue concern.

**Comment by alexh** [comment]
Any updates on this?

**Comment by smfr** [comment]
We will mark this proposal as support in a week pending objections. There are some ongoing discussions around parts of the API shape that we'd like to see continue.
